fixture-monkey-junit-jupiter/src/main/java/com/navercorp/fixturemonkey/junit/jupiter/annotation/ShowMeLog.java

```diff
@@ -0,0 +1,12 @@
+package com.navercorp.fixturemonkey.junit.jupiter.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD})
+public @interface ShowMeLog {
+	String value() default "";
+}
```

Comment:
@codesejin 
I think it would be better to add a test to show the fixed seed result.

Better to add a comment that `value()` is for seeds, or how about renaming `@ShowMeLog` to `@Seed`.

Reply:
Thank you so much for your feedback. To show the fixed seed result, the test would need to fail. Are you suggesting that we should include checking the log output from the test failure in the test logic?😅